client/src/App.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import './App.css';
-import CreatePokemon from './pages/CreatePokemon';
 import { BrowserRouter as Router, Redirect, Route, Switch } from 'react-router-dom';
+import CreatePokemon from './pages/CreatePokemon';
+import ViewPokemons from './pages/ViewPokemons';
 
```

Comment:
super small nitpick but plural of pokemon is still pokemon lmao so can you just change Pokemons to Pokemon? you should be able to just find and replace in your editor easily